airflow/api/common/experimental/mark_tasks.py

```diff
@@ -37,22 +37,23 @@ def _create_dagruns(dag, execution_dates, state, run_id_template):
     :return: newly created and existing dag runs for the execution dates supplied
     """
     # find out if we need to create any dag runs
-    drs = DagRun.find(dag_id=dag.dag_id, execution_date=execution_dates)
-    dates_to_create = list(set(execution_dates) - set([dr.execution_date for dr in drs]))
+    dag_runs = DagRun.find(dag_id=dag.dag_id, execution_date=execution_dates)
+    dates_to_create = list(set(execution_dates) - {dag_run.execution_date for dag_run in dag_runs})
 
     for date in dates_to_create:
-        dr = dag.create_dagrun(
+        dag_run = dag.create_dagrun(
             run_id=run_id_template.format(date.isoformat()),
             execution_date=date,
             start_date=timezone.utcnow(),
             external_trigger=False,
             state=state,
         )
-        drs.append(dr)
-
-    return drs
+        dag_runs.append(dag_run)
 
+    return dag_runs
 
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-locals
```

Comment:
You could also write this as a one-liner: `# pylint: disable=too-many-arguments,too-many-locals`